front/src/applications/editor/tools/rangeEdition/catenary/CatenaryEditionLayers.tsx

```diff
@@ -185,7 +186,7 @@ export const CatenaryEditionLayers: FC = () => {
     <>
       <GeoJSONs
         colors={colors[mapStyle]}
-        layers={layers}
+        layers={editorState.editorLayers}
```

Comment:
If you use the firs comment 
```suggestion
        layers={editorLayers}
```